infrastructure-playbooks/switch-from-non-containerized-to-containerized-ceph-daemons.yml

```diff
@@ -38,12 +38,18 @@
 
   become: true
 
+  vars:
+    delegate_facts_host: True
+
   tasks:
     - name: gather and delegate facts
       setup:
       delegate_to: "{{ item }}"
       delegate_facts: True
-      with_items: "{{ groups['all'] }}"
+      with_items: "{{ groups['all'] | difference(groups.get('clients', [])) }}"
```

Comment:
should we import ceph-defaults role first and change `clients` to `client_group_name` ?

Reply:
I think so